src/gluonts/evaluation/_base.py

```diff
@@ -554,4 +625,21 @@ def __call__(
             all_agg_metrics, all_metrics_per_ts
         )
 
+        if len(self.target_agg_funcs) != 0:
```

Comment:
minor: one can check if the `dict` is not empty as follows (maybe also better readable):
```
if self.target_agg_funcs:
```

Reply:
Nice! Thank you!